fact/src/event.rs

```diff
@@ -12,7 +13,7 @@ use crate::{
     host_info,
 };
 
-fn slice_to_string(s: &[i8]) -> anyhow::Result<String> {
+fn slice_to_string(s: &[c_char]) -> anyhow::Result<String> {
```

Comment:
Is this here because the C char representation is different on x86 and Arm? TIL

Reply:
Yeah this was a bit of a surprise for me too. It's unsigned on ARM so the compiler was complaining about it.